R/linter_tags.R

```diff
@@ -157,17 +157,26 @@ rd_taglist <- function() {
 #' @noRd
 rd_linterlist <- function() {
   linters <- available_linters()
-  linter_names <- sort(linters[["linter"]])
+  linter_names <- platform_independent_sort(linters[["linter"]])
 
   c(
     "\\section{Linters}{",
     "The following linters exist:",
     "\\itemize{",
     vapply(linter_names, function(linter_name) {
-      tags <- sort(linters[["tags"]][[match(linter_name, linters[["linter"]])]])
+      tags <- platform_independent_sort(linters[["tags"]][[match(linter_name, linters[["linter"]])]])
       paste0("\\item{\\code{\\link{", linter_name, "}} (tags: ", toString(tags), ")}")
     }, character(1L)),
     "}", # itemize
     "}" # section
   )
 }
+
+platform_independent_sort <- function(x) {
+  if (!identical(sort(c("a", "B")), c("a", "B"))) {
```

Comment:
I figure this is "good enough" for the purposes of what we want here -- if a dev wants to generate docs, just make sure their locale is "english enough".